docker/datanode/README.md

```diff
@@ -1,9 +1,52 @@
-Graylog Data Node Image
-=======================
+# Graylog Data Node Docker Image
 
-tbd
+[![Docker Stars](https://img.shields.io/docker/stars/graylog/graylog.svg)][hub] [![Docker Pulls](https://img.shields.io/docker/pulls/graylog/graylog.svg)][hub]
```

Comment:
This badge shows the stars for the graylog/graylog image and not graylog/graylog-datanode.